### ISSUE TITLE
Pin V3 Requirements/Dependancies to specific versions

### ISSUE THREAD

**Issue by jmunowitch** [archesproject/arches#728]
These are the versions of the pip installed libraries that we know work with V3. 

```
Django==1.6.2
elasticsearch==1.4.0
html5lib==0.999
isodate==0.5.1
Pillow==2.4.0
psycopg2==2.5.4
pyparsing==2.0.3
pyshp==1.2.1
PyYAML==3.11
rdflib==4.1.2
six==1.9.0
SPARQLWrapper==1.6.4
unicodecsv==0.9.4
urllib3==1.10.1
xlrd==0.9.0
```

**Comment by apeters** [comment]
is this done?

**Comment by jmunowitch** [comment]
It is.

**Comment by mradamcox** [comment]
For posterity, I'll mention that I've just figured out that on Oracle Linux 6.7, this version of Pillow doesn't work. The issue is the jpeg decoder (see [this forum post](https://groups.google.com/forum/#!searchin/archesproject/jpeg%7Csort:relevance/archesproject/oTuxb5em3jY/ithADqhyAgAJ)). I tried installing all the extra yum packages and stuff, but still no luck. However, upgrade to the latest Pillow, 4.1.1, seems to work so far.
